flink-yarn/src/test/java/org/apache/flink/yarn/YarnFileStageTest.java

```diff
@@ -261,7 +263,8 @@ private static void testCopySingleFileFromLocal(
 				remotePaths,
 				localResources,
 				localResourceDirectory,
-				new StringBuilder());
+				new StringBuilder(),
+				DFSConfigKeys.DFS_REPLICATION_DEFAULT);
```

Comment:
By default, we only start 1 datanode. So it is not reasonable to set to `DFSConfigKeys.DFS_REPLICATION_DEFAULT` which value is 3 here.

Reply:
I just to make it use default value here. I think we can still create multiple file replications on single datanode.

Reply:
No, we could not create multiple file replication on single datanode. The file will not be accessible until enough datanode has been registered.

Reply:
@wangyang0918 given this settings to 1, is this patch looks good to you?

Reply:
Though we don't access the file so the test pass, it seems we start a mini HDFS cluster which has one datanode so that set to 1 is more expressive?

Reply:
I think my argument is a little bit misleading. For real HDFS cluster, name node will prevent more than one replica of a block from been placed to the same data node. From my debugging experience, there is no such limitation for MiniHDFSCluster.